packages/ketcher-react/src/script/index.js

```diff
@@ -74,19 +73,14 @@ function showMolfile(clientArea, molString, options) {
   return render
 }
 
-function createDefaultStructService(baseUrl, defaultOptions) {
-  const service = new RemoteStructService(baseUrl, defaultOptions)
-  return service
-}
-
 // TODO: replace window.onload with something like <https://github.com/ded/domready>
 // to start early
-export default function init(el, staticResourcesUrl, apiPath, structServiceFn) {
-  ketcher.apiPath = apiPath
-  const params = new URLSearchParams(document.location.search)
-  const createStructServiceFn = structServiceFn || createDefaultStructService
-  if (params.has('api_path')) ketcher.apiPath = params.get('api_path')
```

Comment:
Please do not remove api_path initialization which is based on query params

Reply:
reverted back